agent/agent.go

```diff
@@ -48,7 +49,6 @@ type Agent struct {
 	config          *Config
 	connected       bool
 	connectedMu     sync.RWMutex
-	context         context.Context
```

Comment:
Is the `cancel` field above still needed?

Reply:
Nope, I'll remove it